README.md

```diff
@@ -284,20 +284,15 @@ To get help, follow the instructions in the [Troubleshooting document](https://g
 
 ## Configuring a Proxy
 
-### HTTP Clients
```

Comment:
Note: with Eric's explanation, now gRPC respects `http.proxyHost` (the same way as HTTP clients). So no need to have separate sections for HTTP and gRPC.